docs/quickstart-overview.md

```diff
@@ -1,35 +1,48 @@
 
-* [Use binary tarballs](binary-tarball.md)
+You are welcome to change the names of any items in commands. If you do, the results may vary from those we provide in this Quickstart guide.
 
-* [Use Docker](docker.md)
+In this section, container refers to the Docker container and instance refers to the database server in the container.
 
-## For superior and optimized performance
+## Prerequisites
 
-Percona Server for MySQL (PS) is a freely available, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It provides superior and optimized performance, greater scalability and availability, enhanced backups, increased visibility, and instrumentation. Percona Server for MySQL is trusted by thousands of enterprises to provide better performance and concurrency for their most demanding workloads.
+* [Install Docker](https://docs.docker.com/engine/install/) on your system.
 
-Install [Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/installation.html).
+To take a backup of Percona Server for MySQL, you need to start Percona Server for MySQL in a docker container and create a database, and a table in Percona Server for MySQL.
 
-## For high availability
+* [Start Percona Server in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.html)
+* [Create a database and table in Percona Server](https://docs.percona.com/percona-server/8.0/quickstart-database.html)
 
-Percona XtraDB Cluster (PXC) is a 100% open source, enterprise-grade, highly available clustering solution for MySQL multi-master setups based on Galera. PXC helps enterprises minimize unexpected downtime and data loss, reduce costs, and improve performance and scalability of your database environments supporting your critical business applications in the most demanding public, private, and hybrid cloud environments. 
+### Supported storage engines
 
-Install [Percona XtraDB Cluster](https://docs.percona.com/percona-xtradb-cluster/8.0/install/index.html).
+Percona XtraBackup 8.0 can back up data from InnoDB, XtraDB, MyISAM, and MyRocks tables on MySQL servers as well as Percona Server for MySQL with XtraDB, Percona Server for MySQL, and Percona XtraDB Cluster.
 
-## For Monitoring and Management
+### Limitations
 
-Percona Monitoring and Management (PMM) monitors and provides actionable performance data for MySQL variants, including Percona Server for MySQL, Percona XtraDB Cluster, Oracle MySQL Community Edition, Oracle MySQL Enterprise Edition, and MariaDB. PMM captures metrics and data for the InnoDB, XtraDB, and MyRocks storage engines, and has specialized dashboards for specific engine details.
+Percona XtraBackup 8.0 does not support making backups of databases created in versions before 8.0 of MySQL, Percona Server for MySQL or Percona XtraDB Cluster.
```

Comment:
or versions greater than 8.0, such as the Innovation releases, 8.1 and 8.2.

Reply:
added